### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in 686/686c-674 end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7315]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/disability-benefits/686` and `/disability-benefits/686c-674` app folders, I wasn't sure if there were modals or hidden content that needed axe checks. I'd like the front-end engineering team to review this application, and add end-to-end tests as needed. Definition of done in acceptance criteria below.

In consult with Jason and Micah, it's possible these are versions of the same application, so I've bundled the review together. Please review the current version and discard the older one.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## Environment
* `vets-website/src/applications//disability-benefits/686/tests`
* `vets-website/src/applications//disability-benefits/686c-674/tests`
* `$ yarn test:e2e src/applications//disability-benefits/686/tests`
* `$ yarn test:e2e src/applications//disability-benefits/686c-674/tests`

## Acceptance Criteria

**Definition of done:**
- [x] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [x] axe checks are run for hidden content like modal windows, accordions
- [x] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [x] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by jason-gcio** [comment]
@jenstrickland - I mistook this ticket for something else but I believe these items are all done.  Would you like to close this or can I?

**Comment by jenstrickland** [comment]
Sure. We'll evaluate on a case-by-case basis to ensure work continues to follow this guidance (especially where hidden and dynamic content needs to be exposed to the axe e2e checks.
